docs/admin_api/user_admin_api.md

```diff
@@ -1013,3 +1013,16 @@ The following parameters should be set in the URL:
 - `user_id` - The fully qualified MXID: for example, `@user:server.com`. The user must
   be local.
 
+### Check username availability
```

Comment:
please can you define what "available" actually means here?

Reply:
Or rather, I guess, give a short summary and link to the `/register/available` API for more details.

Reply:
I've added a short description, there is a link already here to the existing docs further down.

Reply:
the link further down just talks about the request/response format shape. It doesn't say anything about defining what "available" actually means. I'm just talking about a "see ... for more information" link. See the suggestion below.